vsc/storage.py

```diff
@@ -39,7 +39,7 @@ def same_value_ranges(values):
     yield value, start, len(values)
 
 
-def load_features(f, dataset: Optional[Dataset] = None):
+def load_features(f, dataset: Optional[Dataset] = None, as_dict: bool = False):
```

Comment:
Could we keep this method as is, and add a new function that translates `List[VideoFeature]` to a dict? I'd rather have a simple signature for this function.

Reply:
Sure, I have added a new function in the same .py to convert `List[VideoFeature]` to a dict, as suggested.

Reply:
Great. I believe that change isn't reflected in the PR yet, but perhaps I've missed it